config/default.schema.json

```diff
@@ -8,7 +8,12 @@
 		},
 		"NODE_ENV": {
 			"type": "string",
-			"enum": ["development", "test", "production", "migration"],
+			"enum": [
+				"development",
+				"test",
+				"production",
+				"migration"
+			],
```

Comment:
Warum ändert sich den die Formatierung der Array und der anderen Arrays in dieser Datei?